middlewares/validators/user.js

```diff
@@ -26,13 +26,21 @@ const updateUser = async (req, res, next) => {
       company: joi.string().optional(),
       designation: joi.string().optional(),
       img: joi.string().optional(),
-      linkedin_id: joi.string().optional(),
+      linkedin_id: joi
+        .string()
+        .optional()
+        .regex(/^[^@\s]*$/)
+        .message("Invalid Linkedin ID. ID should not contain special character @ or spaces"),
       twitter_id: joi
         .string()
         .optional()
-        .regex(/^[^@]*$/)
-        .message("Invalid Twitter ID. ID should not contain special character @"),
-      instagram_id: joi.string().optional(),
+        .regex(/^[^@\s]*$/)
+        .message("Invalid Twitter ID. ID should not contain special character @ or spaces"),
+      instagram_id: joi
+        .string()
+        .optional()
+        .regex(/^[^@\s]*$/)
+        .message("Invalid Instagram ID. ID should not contain special character @ or spaces"),
```

Comment:
Nit: Would it be possible to create a custom function for this check?

Reply:
These are the methods that joi validation provides which are inline i.e. they are applied on routes or validators only, another function would not be a good idea here.